### PR TITLE
Fix Unit Test

### PR DESCRIPTION
**Pull request by ArneSchulze**
- Random removed
- Coverage raised

### CODE REVIEW

File: app/src/test/java/de/ameyering/wgplaner/wgplaner/MoneyUnitTest.java [hunk 1]
```diff
@@ -5,19 +5,30 @@
...
 
 @RunWith(MockitoJUnitRunner.class)
 public class MoneyUnitTest {
 
+
     @Test
-    public void testToString() {
-        Random random = new Random();
+    public void testConstructor(){
+        for(int i = -2; i < 101; i++){
```
Comment (archer96) [inline app/src/test/java/de/ameyering/wgplaner/wgplaner/MoneyUnitTest.java]: It's better to only test the edge cases.

File: app/src/test/java/de/ameyering/wgplaner/wgplaner/MoneyUnitTest.java [hunk 2]
```diff
@@ -45,6 +51,27 @@ public void testToString() {
                 Assert.assertEquals(expected, test.toString());
             }
         }
+        Locale locale = new Locale("de", "DE");
+        Money.initialize(locale);
+
+        for (int i = 0; i < 100; i++) {
```
Comment (archer96) [inline app/src/test/java/de/ameyering/wgplaner/wgplaner/MoneyUnitTest.java]: Again, test for edge cases only. Having this nested logic makes tests less clean.

File: app/src/test/java/de/ameyering/wgplaner/wgplaner/MoneyUnitTest.java (hunk 1 above)
Comment (ArneSchulze) [reply app/src/test/java/de/ameyering/wgplaner/wgplaner/MoneyUnitTest.java]: Done

File: app/src/test/java/de/ameyering/wgplaner/wgplaner/MoneyUnitTest.java (hunk 2 above)
Comment (ArneSchulze) [reply app/src/test/java/de/ameyering/wgplaner/wgplaner/MoneyUnitTest.java]: Done
